utility/get_ip_addrs.py

```diff
@@ -0,0 +1,57 @@
+""" Look at Ceph storage pods, return ip-addresses """
+import subprocess
+import shlex
+
+
+def get_ip_addrs_for_ceph_type(pod_type, namespace='openshift-storage'):
+    """
+    Get the ip address for all pods of a given type.
+    Args:
```

Comment:
Add empty before this line.
General note: separate sections in docstring with an empty line:
```python
"""
Some description

Args:
    arg1 (str): etc.

Returns:
    bool: etc...
"""

```

Reply:
and empty line before ending \"\"\"

Reply:
change will be made